scripts/artifact-building/amphora-image/rpc-build-image.yml

```diff
@@ -40,15 +39,12 @@
     - name: Create Octavia tmp dir
       file:
         state: directory
-        path: "/var/lib/octavia"
-    - name: Set Octavia tmp dir
-      set_fact:
-        bootstrap_host_octavia_tmp: "/var/lib/octavia"
+        path: "{{ bootstrap_host_octavia_tmp }}"
     - name: Install pip requirements
       pip:
         name: "{{ item }}"
         state: "present"
-        extra_args: "-c https://git.openstack.org/cgit/openstack/requirements/plain/upper-constraints.txt?id={{ requirements_git_install_branch | regex_replace(' #.*$','') }}"
+        extra_args: "-c https://git.openstack.org/cgit/openstack/requirements/plain/upper-constraints.txt?id={{ requirements_git_install_branch | regex_replace(' #.*$','') }} --isolated"
```

Comment:
Why ``--isolated``? The host this runs on will not have any pip configuration applied.